### DISCUSSION TITLE
Custom module not working properly

### DISCUSSION THREAD

**Discussion by Piyush1321** [synthetichealth/synthea#1374] [Q&A]
n the custom module I've written, I've defined two distributed transition states: state1 with a distribution of 0.6 and state2 with a distribution of 0.4. However, when I run the code for, let's say, 10 patients, the number of state1 and state2 instances doesn't seem to be consistent. It should ideally be 6 instances of state1 and 4 instances of state2 for 10 patients, but that's not what I'm observing. Can someone please help me identify what I might be doing wrong?
[testing_state.json](https://github.com/synthetichealth/synthea/files/13050530/testing_state.json)

**Comment by dehall** [comment]
What results are you seeing? Distributed transitions are random so you are not guaranteed to get the exact ratio in the output, but as you run more instances the observed ratio will approach what's defined in the transition. 10 is a fairly small number in terms of randomness, so as long as you aren't consistently seeing all 10 go down one path it's probably just how the randomness is shaking out.

One other thing to be aware of, if you provide the same seed for multiple runs (ex, `run_synthea -s 1234`) you will get the exact same results out in those runs.

**Reply by jawalonoski** [reply]
I agree with what @dehall wrote. Run a 1000 patients, and you'll see a distribution closer to 60:40, but it will never be exact -- that is the nature of probability.
